src/Prefabs/footer.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+echo '<footer>
```

Comment:
this doesnt have to be echo'd - just remove the php tags and have the html code directly in the file